components/classes/conversion.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/conversion",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Conversion",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Class for tracking conversion data like currency conversion",
+  "definitions": {
+    "conversion": {
+      "properties": {}
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/conversion"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/data/record"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/external-source-system-audit-details",
+      "meta:refProperty": [
+        "/extSourceSystemAudit/externalKey",
```

Comment:
We can remove this. "_id " from the record behavior could be used to map the ID for a conversion record